org.snowflake_ssh.App.json

```diff
@@ -0,0 +1,59 @@
+{
+	"app-id": "org.snowflake_ssh.App",
+	"runtime": "org.freedesktop.Platform",
+	"runtime-version": "23.08",
+	"sdk": "org.freedesktop.Sdk",
+	"sdk-extensions": [
+		"org.freedesktop.Sdk.Extension.openjdk11"
+	],
+	"command": "snowflake",
+	"modules": [
+		{
+			"name": "openjdk",
+			"buildsystem": "simple",
+			"build-commands": [
+				"/usr/lib/sdk/openjdk11/install.sh"
+			]
+		},
+		{
+			"name": "snowflake",
+			"buildsystem": "simple",
+			"build-commands": [
+				"mkdir -p /app/snowflake",
+				"mkdir -p /app/bin",
+				"mkdir -p /app/share/icons/hicolor/128x128/apps",
+				"mkdir -p /app/share/applications",
+				"cp snowflake.jar /app/snowflake",
+				"cp flatpak/snowflake /app/bin",
+				"chmod 755 /app/bin/snowflake",
+				"mv snowflake.png org.snowflake_ssh.App.png",
+				"cp org.snowflake_ssh.App.png /app/share/icons/hicolor/128x128/apps",
+				"cp flatpak/org.snowflake_ssh.App.desktop /app/share/applications"
+			],
+			"sources": [
+				{
+					"type": "file",
+					"url": "https://github.com/subhra74/snowflake/releases/download/v1.0.4/snowflake.jar",
+					"sha256": "cc1768e5ce7038d26a893e00066e4d8bed2f42c1ba75b960b324471c5eab74c1"
+				},
+				{
+					"type": "file",
+					"url": "https://dashboard.snapcraft.io/site_media/appmedia/2020/02/snowflake.png",
```

Comment:
why isn't this coming from upstream instead?

Reply:
Because have unsupported size, 512x514.

Reply:
it should be fixed upstream. It's not a flatpak specific issue.